src/runtime/composables/content.ts

```diff
@@ -71,12 +71,12 @@ export const useContent = () => {
   /**
    * Next page from `surround`.
    */
-  const next = computed(() => surround.value?.[1])
+  const next = computed(() => surround?.value[1])
 
   /**
    * Previous page from `surround`.
    */
-  const prev = computed(() => surround.value?.[0])
+  const prev = computed(() => surround?.value[0])
```

Comment:
```suggestion
  const prev = computed(() => surround.value?.[0])
```

surround is a ref, and we should check its value for `undefined`